pkg/api/source/grpc_adapter.go

```diff
@@ -49,14 +49,24 @@ type (
 		// Migration path:
 		//
 		//	Old approach:
-		//	  return executor.StreamOutput{
-		//	  	Output: out,
+		//	  out := source.StreamOutput{
+		//	  	Output: make(chan []byte),
 		//	  }
+		//	  go func() {
+		//	  	out.Output <- []byte("Ticker Event")
+		//	  }()
+		//	  return out, nil
 		//
 		//	New approach:
-		//	  return executor.StreamOutput{
-		//	  	Message: api.NewPlaintextMessage(out, true),
+		//	  out := source.StreamOutput{
+		//	  	Event: make(chan source.Event),
 		//	  }
+		//	  go func() {
+		//	  	out.Event <- source.Event{
+		//	  		Message: api.NewPlaintextMessage("Ticker Event", true),
+		//	  	}
+		//	  }()
+		//	  return out, nil
 		Output chan []byte
```

Comment:
Is there any reason of still keeping this?

Reply:
that's a good question. We have 2 options and I don't have a strong opinion:
1. Mark it as deprecated and remove it in next release
2. Remove it already and change all related plugins

We need to do that for executor and source. 

Personally, I would state with the 1st option just to get used to at least one release deprecation policy.

@huseyinbabal @pkosiec WDYT?

Reply:
Better to handle later then 🙂